teams/docs-rs.toml

```diff
@@ -44,3 +44,6 @@ address = "help@docs.rs"
 
 [[zulip-groups]]
 name = "T-docs-rs"
+
+[[zulip-streams]]
```

Comment:
It works a bit differently. The channel should only be included in one file (here), but it should have:
```toml
extra-teams = [
    "infra",
]
```